Web/Types.hs

```diff
@@ -1,58 +1,62 @@
 module Web.Types where
 
-import IHP.Prelude
-import IHP.ModelSupport
 import Generated.Types
+import IHP.ModelSupport
+import IHP.Prelude
 
 -- Custom types
 
-{-| The result of fetching Landing page with all its paragraphs.
-
-@see `fetchLandingPageWithRecords`
--}
+-- | The result of fetching Landing page with all its paragraphs.
+--
+-- @see `fetchLandingPageWithRecords`
 data LandingPageWithRecords = LandingPageWithRecords
-    { landingPage :: !LandingPage
-    , paragraphCtas :: ![ParagraphCta]
-    , paragraphQuotes :: ![ParagraphQuote]
-    , paragraphCtaRefLandingPages :: ![LandingPage]
-    } deriving (Show)
+  { landingPage :: !LandingPage,
+    paragraphCtas :: ![ParagraphCta],
+    paragraphQuotes :: ![ParagraphQuote],
+    paragraphCtaRefLandingPages :: ![LandingPage]
+  }
+  deriving (Show)
+
+instance Record LandingPageWithRecords where
+  newRecord = LandingPageWithRecords newRecord def def def
+
+instance SetField "paragraphCtas" LandingPageWithRecords [ParagraphCta] where
```

Comment:
Why are these needed?

Reply:
I'm using `newRecord` here https://github.com/klarkc/gizra/blob/1eb53effa3c64a0a30e53f33a1389a77274f5b72/tests/Test/Web/View/LandingPages/ShowSpec.hs#L34-L59, using `newRecord` demands these instances, but I can build the record without these if needed.